debian/clearwater-nginx.postinst

```diff
@@ -59,7 +59,6 @@ SRC_CONF_FILE=$SRC_CONF_DIR/nginx.monit
 case "$1" in
     configure)
       install $SRC_CONF_FILE $MONIT_CONF_FILE
-      sudo service nginx start
       # Remove the default config and enable the ping server.
       rm -f /etc/nginx/sites-enabled/default
       rm -f /etc/nginx/sites-available/default
```

Comment:
We should **stop** nginx after calling `ensite` (reload doesn't always do everything needed).